FFmpegInterop/Source/FFmpegInteropMSS.cpp

```diff
@@ -360,10 +394,12 @@ HRESULT FFmpegInteropMSS::InitFFmpegContext(bool forceAudioDecode, bool forceVid
 			if (avFormatCtx->streams[videoStreamIndex]->disposition == AV_DISPOSITION_ATTACHED_PIC)
 			{
 				videoStreamIndex = AVERROR_STREAM_NOT_FOUND;
+				thumbnailStreamIndex = videoStreamIndex;
```

Comment:
These two lines need to be reversed. ThumbnailStreamIndex will get the AVERROR_STREAM_NOT_FOUND from videoStreamIndex